.github/workflows/unit-tests.yml

```
@@ -0,0 +1,75 @@
+name: UnitTests
+
+on:
+    pull_request:
+    push:
+
+jobs:
+    tests:
+        name: "PHP ${{ matrix.php }} + Composer ${{ matrix.dependency }} + Symfony ${{ matrix.symfony }}"
+        runs-on: ubuntu-latest
+
+        strategy:
+            fail-fast: true
+            matrix:
+                include:
+                    -   php: '5.5'
+                        dependency: lowest
+                        coverage: xdebug
+                        symfony: 'any'
+                    -   php: '5.5'
+                        dependency: highest
+                        coverage: xdebug
+                        symfony: 'any'
+                    -   php: '5.6'
+                        dependency: highest
+                        coverage: xdebug
+                        symfony: 'any'
+                    -   php: '7.0'
+                        dependency: highest
+                        coverage: xdebug
+                        symfony: 'any'
+                    -   php: '7.1'
+                        dependency: highest
+                        coverage: xdebug
+                        symfony: '^3.0'
+                    -   php: '7.2'
+                        dependency: highest
+                        coverage: pcov
+                        symfony: '^4.0'
+                    -   php: '7.3'
+                        dependency: highest
+                        coverage: pcov
+                        symfony: '^5.0'
+                    -   php: '7.4'
+                        dependency: highest
+                        coverage: pcov
+                        symfony: '^5.0'
```

```suggestion
                        symfony: '^6.0'
```

we checked ^5 in previous PHP 7.3 job

---

next step I will extend this matrix. Currently this is moved as is, without changes, to have point to start